terraform-aws-github-runner/modules/runners/lambdas/runners/README.md

```diff
@@ -0,0 +1,164 @@
+# Runners Autoscaler Lambda Infrastucture - Typescript code
+
+This folder contains the typescript code for the scaleUp and scaleDown lambdas
+
+## Local Development
+
+### Requirements
+
+Node, pip, yarn, cmake
+
+### Development main workflow
+
+Most of the development is done using `yarn`, in order to setup the project, run:
+
+```
+$ yarn install
+```
+
+Next, you can run unit test by:
+
+```
+$ yarn test
+```
+
+after completing your changes, please run:
+
+```
+$ yarn commit-check
+```
+
+This should lint the code, format it, make sure it is building properly and run tests. If your command succeeds, it will be green on CI.
+
+### Expectations
+
+It is **required** to submit code:
+
+* All unit tests are passing;
+* At least 80% unit test coverage;
+* No linting warnings are being thrown;
+* The code must be fully compilable to javascript;
+* Code formatting are according to current standards (prettier);
```

Comment:
Doesn't CI fail if these requirements aren't met?

Can we instead change these three lines to: "All CI checks pass"